prometheus/base/prometheus.rules.yaml

```diff
@@ -15,3 +15,22 @@ spec:
             * on (namespace, pod)
             group_left (label_pipelines_appstudio_openshift_io_type)
             kube_pod_labels
+        # containers and init containers resource limits with added pod labels
+        - record: appstudio_container_resource_limits_with_additional_labels
+          expr: |
+            {__name__=~"kube_pod_.*container_resource_limits"}
+            * on (namespace, pod)
+            group_left(label_pipelines_appstudio_openshift_io_type) kube_pod_labels
+        - record: labeled_containers:resource_minutes:2w
+          expr: |
+            last_over_time(appstudio_container_resource_limits_with_additional_labels[2w])
+            * on (namespace, pod, container) group_left ()
+            (sum by (namespace, pod, container) (increase(container_last_seen[2w] )) / 60)
+    - name: resource_gauge
+      interval: 5m
+      rules:
+        - record: appstudio_container_resource_minutes_gauge
```

Comment:
```suggestion
        - record: appstudio_container_resource_5minutes_gauge
```

Reply:
I'm a bit reluctant about including the window size here as consumer should not care about this.

Reply:
In this case, remove the `minutes`  as it has no meaning here, maybe `appstudio_container_resource_gauge`

Reply:
It's still resource minutes gauge. To say, the units of the value it records is resource/minute.

Reply:
OK, I would however add `limits` somewhere,  `appstudio_container_resource_limits_minutes_gauge` since we are using the `kube_pod_container_resource_limits`.
Just not to get confused by the name with `kube_pod_container_resource_requests`

Reply:
Good point. I'll add it.

Reply:
On another thought, the rule is meant to represent resource consumption minutes. Using the limit is just how we decided to address it. If tomorrow we change it to be based on some other metric, I think we might want to keep the same name and just change the implementation.

For that reason, I prefer to leave the name as-is.

Reply:
So lets keep it for now, we can always change it if it would be needed